code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -157,40 +186,61 @@
 			owner.hud_used.move_intent.icon_state = "running"
 		owner.update_icons()
 
-	cancel_stealth()
+	if(pounce_break_stealth)
+		cancel_stealth()
 
 /// Callback for when a mob gets hit as part of a pounce
 /datum/action/ability/xeno_action/stealth/proc/mob_hit(datum/source, mob/living/M)
 	SIGNAL_HANDLER
 	if(M.stat || isxeno(M))
 		return
 	if(can_sneak_attack)
-		M.adjust_stagger(3 SECONDS)
-		M.add_slowdown(1)
+		var/datum/action/ability/activable/xeno/hunter_mark/assassin/mark = owner.actions_by_path[/datum/action/ability/activable/xeno/hunter_mark/assassin]
+		if(mark.marked_target == M)
+			to_chat(owner, span_xenodanger("We strike our death mark with a calculated pounce."))
+			M.adjust_stagger(6 SECONDS)
+			M.add_slowdown(2)
+		else
+			M.adjust_stagger(3 SECONDS)
+			M.add_slowdown(1)
 		to_chat(owner, span_xenodanger("Pouncing from the shadows, we stagger our victim."))
+	if(pounce_hit_break_stealth)
+		cancel_stealth()
+	else
+		can_sneak_attack = FALSE
+		addtimer(CALLBACK(src, PROC_REF(sneak_attack_cooldown)), HUNTER_POUNCE_SNEAKATTACK_DELAY)
 
 ///Special sneak attack when stealthed
 /datum/action/ability/xeno_action/stealth/proc/sneak_attack_slash(datum/source, mob/living/target, damage, list/damage_mod, list/armor_mod)
 	SIGNAL_HANDLER
 	if(!can_sneak_attack)
 		return
 
-	var/staggerslow_stacks = 2
+	var/staggerslow_stacks = 2 SECONDS
```

Comment:
why are you multiplying the slowdown by 10??

Reply:
ah shit you are right, someone mentioned about why i used just 2 in the var itself then seconds at the actual effect adding and that got me confused too is the story id say, ill fix

Reply:
mistake i think

Reply:
should be fixed now, also got rid of the seemingly unused walk/run thing i guess.

Reply:
How is that unused? You're just removing features